decks/Elections-Proposal.mdx

```diff
@@ -0,0 +1,65 @@
+import { CodeSurfer as Surfer } from "code-surfer";
+import { CodeSurferColumns, Step } from "code-surfer";
+import { Appear, Background } from "gatsby-theme-mdx-deck";
+import * as L from "../src/layout";
+import customTheme from "../src/theme";
+import GreetingLoader from "./src/greeting-loader";
+
+export const theme = customTheme;
+
+# Workshop Proposal
+
+---
+
+# Specific Topic
+- We are going to use the OpenFEC API Election endpoint to visualize data for total disbursements and total receipts for Presidential candidates for the last 30 elections.
+- Graph will compare this financial information over time and for both the Democratic and Republican parties. 
+
+---
+
+# Interactive Component
+- We will have students visualize data from the same endpoint but for cash_on_hand_end_period 
+---
```

Comment:
practice how to import images as well. it could help someone as myself better understand what OpenFEC is